presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -28,6 +28,7 @@
 import javax.inject.Inject;
```

Comment:
Please extend the commit message. Explain that it is about CBO and explain the problem and when it should be used.